cmd/gobitcask/commands/root.go

```
@@ -0,0 +1,36 @@
+	// Add subcommands
+	rootCmd.AddCommand(newPutCommand())
+	rootCmd.AddCommand(newGetCommand())
+	rootCmd.AddCommand(newDeleteCommand())
+	rootCmd.AddCommand(newListCommand())
+	rootCmd.AddCommand(newClearCommand())
+
+	return rootCmd
+}
+
+// printError prints an error message and exits
+func printError(err error) {
```

golangci-lint (1.64.8)

func `printError` is unused

(unused)